assets/js/base/context/providers/add-to-cart-form/form-state/index.js

```diff
@@ -206,17 +206,31 @@ export const AddToCartFormStateContextProvider = ( {
 	 */
 	useEffect( () => {
 		if ( addToCartFormState.status === STATUS.AFTER_PROCESSING ) {
+			// @todo: This data package differs from what is passed through in
+			// the checkout state context. Should we introduce a "context"
+			// property in the data package for this emitted event so that
+			// observers are able to know what context the event is firing in?
```

Comment:
I think this context provider was preliminary work for the add-to-cart form element block which is currently not released. So I've excluded it from testing notes. It's possible this will be revisited when we resume work on that block.